dora/minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -168,10 +168,14 @@ public void startWorkers() throws Exception {
         } catch (InterruptedException e) {
           // this is expected
         } catch (Exception e) {
-          // Log the exception as the RuntimeException will be caught and handled silently by
-          // JUnit
-          LOG.error("Start worker error", e);
-          throw new RuntimeException(e + " \n Start Worker Error \n" + e.getMessage(), e);
+          if (e instanceof RuntimeException && e.getCause() instanceof InterruptedException) {
+            // Expected; No-op
```

Comment:
can we find out which test is wrapping an `InterruptedException` inside a `RuntimeException`? instead of examining the cause inside the test harness.